hugegraph-llm/src/hugegraph_llm/utils/gradio_demo.py

```diff
@@ -146,110 +146,113 @@ def init_config(
-    - Commit to hugegraph: Whether to commit the constructed knowledge graph to the 
-    HugeGraph server.
-    """
-    )
-    TEXT = (
-        "Meet Sarah, a 30-year-old attorney, and her roommate, James, whom she's shared a home with"
-        " since 2010. James, in his professional life, works as a journalist. Additionally, Sarah"
-        " is the proud owner of the website www.sarahsplace.com, while James manages his own"
-        " webpage, though the specific URL is not mentioned here. These two individuals, Sarah and"
-        " James, have not only forged a strong personal bond as roommates but have also carved out"
-        " their distinctive digital presence through their respective webpages, showcasing their"
-        " varied interests and experiences."
-    )
-
-    SCHEMA = """{
-        "vertices": [
-            {"vertex_label": "person", "properties": ["name", "age", "occupation"]},
-            {"vertex_label": "webpage", "properties": ["name", "url"]}
-        ],
-        "edges": [
-            {
-                "edge_label": "roommate",
-                "source_vertex_label": "person",
-                "target_vertex_label": "person",
-                "properties": {}
-            }
-        ]
-    }
-    """
-
-    with gr.Row():
-        inp = [
-            gr.Textbox(value=TEXT, label="Text"),
-            gr.Textbox(value=SCHEMA, label="Schema"),
-            gr.Textbox(value="false", label="Disambiguate word sense"),
-            gr.Textbox(value="false", label="Commit to hugegraph"),
-        ]
-    with gr.Row():
-        out = gr.Textbox(label="Output")
-    btn = gr.Button("Build knowledge graph")
-    btn.click(fn=build_kg, inputs=inp, outputs=out)  # pylint: disable=no-member
-
-    gr.Markdown("""## 2. Retrieval augmented generation by hugegraph""")
-    with gr.Row():
-        inp = gr.Textbox(value="Tell me about Al Pacino.", label="Question")
-    with gr.Row():
-        out = gr.Textbox(label="Answer")
-    btn = gr.Button("Retrieval augmented generation")
-    btn.click(fn=graph_rag, inputs=inp, outputs=out)  # pylint: disable=no-member
-
-    gr.Markdown("""## 3. Others """)
-    with gr.Row():
-        inp = []
-        out = gr.Textbox(label="Output")
-    btn = gr.Button("Initialize HugeGraph test data")
-    btn.click(fn=init_hg_test_data, inputs=inp, outputs=out)  # pylint: disable=no-member
-
-    with gr.Row():
-        inp = gr.Textbox(value="g.V().limit(10)", label="Gremlin query")
-        out = gr.Textbox(label="Output")
-    btn = gr.Button("Run gremlin query on HugeGraph")
-    btn.click(fn=run_gremlin_query, inputs=inp, outputs=out)  # pylint: disable=no-member
-
 if __name__ == "__main__":
     app = FastAPI()
+    initial_config = Config(section=Constants.HUGEGRAPH_CONFIG)
+    with gr.Blocks() as hugegraph_llm:
+        gr.Markdown(
+            """# HugeGraph LLM Demo
+        1. Set up the HugeGraph server."""
```

Comment:
better to paste to show what changes made.

Reply:
Here I replace the constant value in gr.Textbox with the value of reading config.